index.bs

```diff
@@ -3308,6 +3354,45 @@ interface AudioParam {
 Attributes</h4>
 
 <dl dfn-type=attribute dfn-for="AudioParam">
+	: <dfn>automationRate</dfn>
+	::
+		The automation rate for the {{AudioParam}}.  The
+		default value depends on the actual {{AudioParam}};
+		see the description of the {{AudioParam}} for the
+		default value.
+
+		Some nodes have additional <dfn dfn>automation rate constraints</dfn> as follows:
```

Comment:
Typo: `dfn dfn`

Reply:
No, I think that's right.  Without the second "dfn", I think bikeshed thinks it's part of `AudioParam` idl, but it's really just a plain dfn unassociated with anything. (I think that's what Tab told be about other parts in the spec where we have "dfn dfn".